src/framework/COSListener.m

```diff
@@ -35,9 +35,8 @@ + (void)listenWithRootObject:(id)rootObject; {
 
 
 - (void)setupListener {
-    // Hardcode the port to listen on because of sandboxing
-    NSString *myBundleId = @"WUGMZZ5K46.com.bohemiancoding";
-    NSString *port          = [NSString stringWithFormat:@"%@.JSTalk", myBundleId];
+    NSString *myBundleId = NSBundle.mainBundle.bundleIdentifier;
+    NSString *port = [NSString stringWithFormat:@"%@.JSTalk", myBundleId];
 
```

Comment:
This is the fix, the change was an attempt to fix plugins for the MAS version of Sketch. But broke using sketchtool to run plugins in Sketch. Since plugins are not in Sketch MAS I'm restoring this.